Dockerfile

```diff
@@ -50,8 +47,6 @@ COPY cname.py .
 COPY restart.sh .
 RUN touch ./cnames
 
-RUN pip install mdns-publisher
```

Comment:
I've just tested this locally and it fails with this line missing with the following:

```
forego      | starting cname.1 on port 603400
cname.1     | Traceback (most recent call last):
cname.1     |   File "/usr/src/app/cname.py", line 15, in <module>
cname.1     |     from mpublisher import AvahiPublisher
cnforego      | starting cname.1 on port 603400
cname.1     | Traceback (most recent call last):
cname.1     |   File "/usr/src/app/cname.py", line 15, in <module>
cname.1     |     from mpublisher import AvahiPublisher
cname.1     | ModuleNotFoundError: No module named 'mpublisher'
ame.1     | ModuleNotFoundError: No module named 'mpublisher'
```

We definitely need to install `mdns-publisher`

Reply:
Now I think about it, we need the build tools because the mdns-publisher has native components so they are needed to build it.

Reply:
Ahh sorry, must have fallen victim to the build cache, as rebuilding it again with --no-cache I get the same issue.  I guess the only other option then is to create a builder docker container for it and then copy across.  May just go do that for educational purposes, but I doubt the complexity is worth the ~200MB saved from the image.  But once it's done I'll come back with more info